src/bpf/bpfd-manager.c

```diff
@@ -0,0 +1,489 @@
+/* SPDX-License-Identifier: LGPL-2.1+ */
+
+#include <sys/wait.h>
+
+#include "hexdecoct.h"
+#include "parse-util.h"
+#include "sd-daemon.h"
+#include "sd-messages.h"
```

Comment:
the sd-*.h headers should be included first (as they are the apis we define that are accessible to other projects too, and thus are more "public" then the other headers we have. and place an empty line between them and the other include lines